sky/cli.py

```diff
@@ -1263,14 +1263,15 @@ def exec(
     If ENTRYPOINT points to a valid YAML file, it is read in as the task
     specification. Otherwise, it is interpreted as a bash command.
 
-    \b
     Actions performed by ``sky exec``:
 
-    \b
-    (1) workdir syncing, if:
-      - ENTRYPOINT is a YAML and ``workdir`` is specified inside; or
-      - ENTRYPOINT is a command and flag ``--workdir=<local_path>`` is set.
-    (2) executing the specified task's ``run`` commands / the bash command.
+    1. Workdir syncing, if:
+
+       - ENTRYPOINT is a YAML and ``workdir`` is specified inside; or
+
+       - ENTRYPOINT is a command and flag ``--workdir=<local_path>`` is set.
```

Comment:
even if the ENTRYPOINT is a YAML, if the flag `--workdir` is set, the workdir will still synced right? Should this be:
```
- Flag ``--workdir=<local_path>`` is set; or
- ENTRYPOINT is a YAML and ``workdir`` is specified inside
```

Reply:
Great catch! Fixed.